src/api/spec/lib/backend/connection_helper_spec.rb

```diff
@@ -1,6 +1,8 @@
+class FakeClassWithParsePackageDiff; end # rubocop:disable Lint/EmptyClass
+
 RSpec.describe Backend::ConnectionHelper do
   let(:backend_api_fake_class) do
-    extended_class = double('Fake Class with ParsePackageDiff')
+    extended_class = class_double(FakeClassWithParsePackageDiff)
```

Comment:
Given that the class name is of no particular interest, what about using `Class.new`?

Reply:
It works. :+1: Please take another look.